Composer/packages/client/src/pages/publish/Notifications.tsx

```diff
@@ -117,11 +118,23 @@ export const getSkillPublishedNotificationCardProps = (
     margin: 10px 8px 0 0;
   `;
   return {
-    title: item.status === 200 ? formatMessage('Your skill is ready to be shared!') : '',
+    title: item.status === 200 ? formatMessage('Your bot was exported to a skill') : '',
     description:
       item.status === 200
-        ? formatMessage(
-            'Keep this URL handy to share it with other developers to use in their bot projects. You can find this URL in the project settings tab.'
+        ? formatMessage.rich(
+            'Copy and share this URL with the bots that can connect to this skill. <link>Learn more.</link>',
+            {
+              link: ({ children }) => (
+                <Link
+                  key={'republish-skill'}
+                  href={'https://docs.microsoft.com/en-us/composer/how-to-export-a-skill#republish-the-skill-to-azure'}
```

Comment:
Does this need to be an aka link?

Reply:
Same here. aka.ms link. Check this link as a sample
https://redirectiontool.trafficmanager.net/am/redirection/home?options=host:aka.ms&set=disableAkaUUHourly